icu4c/source/common/cjmlbe.h

```diff
@@ -0,0 +1,126 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef CJMLBREAKENGINE_H
+#define CJMLBREAKENGINE_H
+
+#include "unicode/utypes.h"
+#include "unicode/uniset.h"
+#include "unicode/utext.h"
+
+#include "brkeng.h"
+#include "dictbe.h"
+#include "hash.h"
+#include "uvectr32.h"
+
+
+U_NAMESPACE_BEGIN
+
+#if !UCONFIG_NO_NORMALIZATION
+
+
+/**
+ * A machine learning break engine for the phrase breaking in Chinese and Japanese.
+ */
+class CjMLBreakEngine : public UMemory {
+
+public:
+    /**
+     * Constructor.
+     *
+     * @param fDigitOrOpenPunctuationOrAlphabetSet A UnicodeSet with Digit, open punctuation and alphabet.
+     * @param fClosePunctuationSet A UnicodeSet with close punctuation.
+     * @param status Information on any errors encountered.
+     */
+    CjMLBreakEngine(UnicodeSet &fDigitOrOpenPunctuationOrAlphabetSet, UnicodeSet &fClosePunctuationSet, UErrorCode &status);
```

Comment:
Should fDigitOrOpenPunctuationOrAlphabetSet and fClosePunctuationSet be  "const UnicodeSet& " instead ? why do we pass in that reference without "const" ?